components/Cards/SchoolCard/FrontSchoolCard.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Button from 'components/_common_/Button/Button';
+import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
+import DesktopIcon from 'static/images/icons/FontAwesome/desktop-solid.svg';
+import DevicesIcon from 'static/images/icons/FontAwesome/devices-alt-solid.svg';
+import HomeIcon from 'static/images/icons/FontAwesome/home-solid.svg';
+import PeopleIcon from 'static/images/icons/FontAwesome/users-solid.svg';
+import GIBillApprovedIcon from 'static/images/icons/gi-bill-approved.svg';
+import GIBillUnavailableIcon from 'static/images/icons/gi-bill-unavailable.svg';
+import styles from './FrontSchoolCard.css';
+
+export default class FrontSchoolCard extends Component {
+  static propTypes = {
+    cardFlipCallback: PropTypes.func.isRequired,
+    hasHardwareIncluded: PropTypes.bool.isRequired,
+    hasHousing: PropTypes.bool.isRequired,
+    hasOnline: PropTypes.bool.isRequired,
+    hasOnlyOnline: PropTypes.bool.isRequired,
+    isFullTime: PropTypes.bool.isRequired,
+    locations: PropTypes.arrayOf(
+      PropTypes.shape({
+        city: PropTypes.string,
+        va_accepted: PropTypes.bool.isRequired,
+        state: PropTypes.string,
+      }),
+    ).isRequired,
+    logoSource: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+    website: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const { props } = this;
+
+    return (
+      <>
+        <h5>{props.name}</h5>
+        <img src={props.logoSource} alt="" />
+
+        <div className={styles.interactionsContainer}>
+          <Button>
+            <OutboundLink analyticsEventLabel={`${props.name} | Website`} href={props.website}>
+              Go To Website
+            </OutboundLink>
+          </Button>
+
+          {props.locations.length > 1 ? (
+            <Button
+              analyticsObject={{
+                action: 'Button Selected',
+                category: 'Interactions',
+                label: `${props.name} | Locations`,
+              }}
+              onClick={props.cardFlipCallback}
+            >
+              See Locations
+            </Button>
+          ) : (
+            <span className={styles.singleLocationText}>
+              {props.hasOnlyOnline
+                ? 'Online Only'
+                : `${props.locations[0].city}, ${props.locations[0].state}`}
+            </span>
+          )}
+        </div>
+
+        <hr className={styles.divider} />
+
+        <div className={styles.detailsRow}>
+          {/* GI Bill */}
+          {props.locations.some(location => location.va_accepted) ? (
+            <figure className={`${styles.detailItem} ${styles.active}`}>
```

Comment:
Let's change the `<Badge>` component to use `<figure>` and `<figcaption>` and then use the Badge component here. We can also drop `aria-hidden="true"` from all of these, since they're added automatically now via SVG Loader config.